src/lib/rest/httpRequestSend.cpp

```diff
@@ -406,45 +401,50 @@ int httpRequestSendWithCurl
+  std::string userAgentHeaderName = "User-Agent";
 
-  snprintf(headerUserAgent, sizeof(headerUserAgent), "User-Agent: orion/%s libcurl/%s", versionGet(), curlVersionGet(cvBuf, sizeof(cvBuf)));
-  LM_T(LmtHttpHeaders, ("HTTP-HEADERS: '%s'", headerUserAgent));
-  httpHeaderAdd(&headers, "User-Agent", headerUserAgent, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
+  snprintf(userAgentHeaderValue, sizeof(userAgentHeaderValue), "orion/%s libcurl/%s", versionGet(), curlVersionGet(cvBuf, sizeof(cvBuf)));
+  LM_T(LmtHttpHeaders, ("HTTP-HEADERS: '%s'", (userAgentHeaderName + ": " + userAgentHeaderValue).c_str()));
+  httpHeaderAdd(&headers, userAgentHeaderName, userAgentHeaderValue, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
 
   // ----- Host
-  char headerHost[HTTP_HEADER_HOST_MAX_LENGTH];
+  char hostHeaderValue[HTTP_HEADER_HOST_MAX_LENGTH];
+  std::string hostHeaderName = "Host";
 
-  snprintf(headerHost, sizeof(headerHost), "Host: %s:%d", ip.c_str(), (int) port);
-  LM_T(LmtHttpHeaders, ("HTTP-HEADERS: '%s'", headerHost));
-  httpHeaderAdd(&headers, "Host", headerHost, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
+  snprintf(hostHeaderValue, sizeof(hostHeaderValue), "%s:%d", ip.c_str(), (int) port);
+  LM_T(LmtHttpHeaders, ("HTTP-HEADERS: '%s'", (hostHeaderName + ": " + hostHeaderValue).c_str()));
+  httpHeaderAdd(&headers, hostHeaderName, hostHeaderValue, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
 
   // ----- Tenant
   if (tenant != "")
   {
-    std::string fiwareService = std::string("fiware-service: ") + tenant;
-    httpHeaderAdd(&headers, "fiware-service", fiwareService, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
+    std::string fiwareServiceHeaderName = "fiware-service";
+    std::string fiwareServiceHeaderValue = tenant;
+    httpHeaderAdd(&headers, fiwareServiceHeaderName, fiwareServiceHeaderValue, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
   }
 
   // ----- Service-Path
-  std::string fiwareServicePath = std::string("Fiware-ServicePath: ") + ((servicePath.empty())? "/" : servicePath);
-  httpHeaderAdd(&headers, "Fiware-ServicePath", fiwareServicePath, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
+  std::string fiwareServicePathHeaderName = "Fiware-ServicePath";
+  std::string fiwareServicePathHeaderValue =  (servicePath.empty())? "/" : servicePath;
+  httpHeaderAdd(&headers, fiwareServicePathHeaderName, fiwareServicePathHeaderValue, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
 
   // ----- X-Auth-Token
   if (xauthToken != "")
   {
-    std::string xauthTokenString = std::string("X-Auth-Token: ") + xauthToken;
-    httpHeaderAdd(&headers, "X-Auth-Token", xauthTokenString, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
+    std::string xauthTokenHeaderName = "X-Auth-Token";
+    std::string xauthTokenHeaderValue = xauthToken;
+    httpHeaderAdd(&headers, xauthTokenHeaderName, xauthTokenHeaderValue, &outgoingMsgSize, extraHeaders, usedExtraHeaders);
```

Comment:
We try to indent the variables and the equal sign.
The code is a little easier to read that way.
There are plenty of unindented vars in the PR.
Easy enough to fix.

And, remember, TABs are strictly forbidden!

Reply:
> And, remember, TABs are strictly forbidden!

Yeah, rule M4 https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#m4-indent ;)

Reply:
Fixed in af759bc